README.md

```diff
@@ -73,6 +73,17 @@ loaded_model_from_hub = INCModelForSequenceClassification.from_pretrained(
 
 You can load many more quantized models hosted on the hub under the Intel organization [`here`](https://huggingface.co/Intel).
 
+#### One-click acceleration with launcher
+
+Users can use our Python launcher design to run the Python model code as it is with automatic enabling of the optimization. For example,
+
+```bash
+python -m optimum.intel.neural_compressor.launcher run_glue.py --model_name_or_path bert-base-cased --task_name mrpc --do_eval --output_dir result
```

Comment:
We should remove the `eval_func` by default as it results in errors in for transformers examples

Reply:
That's doable. Would you mind elaborating the "error" you mentioned? Is it a Optimum/INC algorithm-related issue, or does it related to this launcher?